guidelines/index.html

```diff
@@ -2256,31 +2256,31 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
          </section>
          
          
-         <section class="informative" id="security-summary"><div class="header-wrapper"><h3 id="x5-7-security-considerations"><bdi class="secno">5.7 </bdi>Security Considerations</h3><a class="self-link" href="#security-summary" aria-label="Permalink for Section 5.7"></a></div><p><em>この節は規定ではない。</em></p>
+         <section class="informative" id="security-summary"><div class="header-wrapper"><h3 id="x5-7-security-considerations"><bdi class="secno">5.7 </bdi>セキュリティの考慮事項</h3><a class="self-link" href="#security-summary" aria-label="Permalink for Section 5.7"></a></div><p><em>この節は規定ではない。</em></p>
             
             
             
-            <p>Success Criteria within this specification which the Working Group has identified possible implications for security, either by providing protections for end users or which are important for web site providers to take in to consideration when implementing features designed to protect user security, are listed below. This list reflects the current understanding of the Working Group but other Success Criteria may have security implications that the Working Group is not aware of at the time of publishing.</p>
+            <p>この仕様内の達成基準のうち、エンドユーザに保護を提供することによってセキュリティに影響を与える可能性があるとワーキンググループが特定したもの、又は、利用者のセキュリティを保護する目的で設計された機能を実装する際にウェブサイト提供者が考慮することが重要であるものを次に示す。このリストはワーキンググループの現在の理解を反映したものであるが、他の達成基準にも、発行時点でワーキンググループが認識していないセキュリティに関する影響があるかもしれない。</p>
```

Comment:
上と同じでこうなんじゃないかと思います。たぶん。

```suggestion
            <p>この仕様内の達成基準のうち、エンドユーザに保護を提供するか、又は、利用者のセキュリティを保護する目的で設計された機能を実装する際にウェブサイト提供者が考慮することが重要であり、セキュリティに影響を与える可能性があるとワーキンググループが特定したものを次に示す。このリストはワーキンググループの現在の理解を反映したものであるが、他の達成基準にも、発行時点でワーキンググループが認識していないセキュリティに関する影響があるかもしれない。</p>
```